docs/content/guides/reverse-proxy.md

```diff
@@ -114,3 +114,25 @@ Here is an example config snippet:
   Include /etc/letsencrypt/options-ssl-apache.conf
 </VirtualHost>
 ```
+### Traefik
+
+In traefik we have to add the traefik labels and it is also very important: add our domain to "CMD_DOMAIN", "CMD_URL_ADDPORT=false", "CMD_PROTOCOL_USESSL=true".
+
+``` yaml
+  app:
+    # Make sure to use the latest release from https://hedgedoc.org/latest-release
+    image: quay.io/hedgedoc/hedgedoc:1.9.2
+    environment:
+      - CMD_DB_URL=postgres://hedgedoc:password@database:5432/hedgedoc
+      - CMD_DOMAIN=yourdomain.com
+      - CMD_URL_ADDPORT=false
```

Comment:
in my setup this isn't required. i'd stick to what is necessary in an example.

Reply:
CMD_DOMAIN and CMD_URL_ADDPORT are necessary if you have a reverse proxy.
https://docs.hedgedoc.org/guides/reverse-proxy/#configure-asset-link-generation

Reply:
at least not in practice, this is the full extend of my configuration that runs for quiet a while now:

```yaml
services:
  app:
    environment:
      CMD_ALLOW_EMAIL_REGISTER: "false"
      CMD_DB_URL: postgres://hedgedoc:hedgedoc@db:5432/hedgedoc
      CMD_DOMAIN: ${DOMAIN:?err}
      CMD_PROTOCOL_USESSL: "true"
```